api/shared.thrift

```diff
@@ -35,3 +35,17 @@ enum Permission {
   PRODUCT_ACCESS   = 17,        // scope: PRODUCT
   PRODUCT_STORE    = 18         // scope: PRODUCT
 }
+
+/**
+* Status information about the database backend.
+*/
+enum DBStatus {
```

Comment:
- Use full sentences in comments.
- Place OK/NO at the end of enum values: `SCHEMA_MISMATCH_MIGRATION_OK`, `SCHEMA_MISMATCH_MIGRATION_NO`
- Use short enum values: `SCHEMA_INIT_FAILED`
```thrift
enum DBStatus {
  OK,                           // Everything is okay.
  MISSING,                      // Database is missing.
  CONNECT_FAILED,               // Failed to connect to the database.
  SCHEMA_MISMATCH_OK,           // Schema mismatch between the server and the database.
  SCHEMA_MISMATCH_NO,           // No automatic migration is available.
  SCHEMA_MISSING,               // Schema is missing from the database.
  SCHEMA_INIT_FAILED,           // Failed to create initial database schema.
  SCHEMA_UPGRADE_FAILED         // Failed to upgrade schema.
}
```